shapely/_geometry.py

```diff
@@ -731,20 +731,23 @@ def set_precision(geometry, grid_size, mode="valid_output", **kwargs):
 
     By default, geometries use double precision coordinates (grid_size = 0).
 
-    Coordinates will be rounded if a precision grid is less precise than the
-    input geometry. Duplicated vertices will be dropped from lines and
-    polygons for grid sizes greater than 0. Line and polygon geometries may
-    collapse to empty geometries if all vertices are closer together than
-    grid_size. Z values, if present, will not be modified.
-
-    Note: subsequent operations will always be performed in the precision of
-    the geometry with higher precision (smaller "grid_size"). That same
-    precision will be attached to the operation outputs.
-
-    Also note: input geometries should be geometrically valid; unexpected
-    results may occur if input geometries are not.
-
-    Returns None if geometry is None.
+    Coordinates will be rounded if the precision grid specified is less precise than the
```

Comment:
Can you keep this to the original line width? (I know in general the project allows 88 per the formatter, but for docstrings this is 1) not actually enforced by the linter, and 2) generally the docstrings are wrapped below 80 (which IMO improves readability))

Reply:
ok